packages/widget-v2/src/pages/SwapPage/SwapPage.tsx

```diff
@@ -36,6 +37,7 @@ export const SwapPage = () => {
   const { isLoading: isRouteLoading, isError: isRouteError, error: routeError } = useAtomValue(skipRouteAtom);
   const swapFlowSettings = useModal(SwapPageSettings);
   const tokenAndChainSelectorFlow = useModal(TokenAndChainSelectorModal);
+  const selectWalletFlow = useModal(WalletSelectorModal);
```

Comment:
nit: rename these vars to "swapPageSettings, tokenAndChainSelectorModal, walletSelectorModal", let's get rid of the word "flow" when we refactor